terraform/testcases/otlp_metric_mock/otconfig.tpl

```diff
@@ -0,0 +1,29 @@
+extensions:
+  pprof:
+    endpoint: 0.0.0.0:1777
+receivers:
+  otlp:
+    protocols:
+      grpc:
+        endpoint: 0.0.0.0:${grpc_port}
+
+processors:
+  batch:
+
+exporters:
+  logging:
+    loglevel: debug
+  awsemf:
+    region: ${region}
+    endpoint: "https://${mock_endpoint}"
```

Comment:
Does this cause exceptions in the collector?

Reply:
It does not. https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/awsemfexporter has documentation about it, but the awsemf endpoint config allows the user to override the cloudwatch service endpoint. In this case, we're overriding it so the mock backend gets the request.